.changeset/good-jars-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search': minor
+---
+
+add IdentityApi support
```

Comment:
Maybe something like:

Use the `identityApi` to forward authorization headers to the `search-backend`?